web/src/transmission.js

```diff
@@ -141,7 +141,9 @@ export class Transmission extends EventTarget {
           this.setCurrentPopup(new MoveDialog(this, this.remote));
           break;
         case 'show-overflow-menu':
-          {
+          if (this.popup instanceof OverflowMenu) {
+            this.popup.close();
```

Comment:
_(nit)_ I think popup.close() would work here due to the popupCloseListener but for consistency with the rest of the places that replace the popup, `setCurrentPopup(null)` would be slightly better here

```suggestion
            this.setCurrentPopup(null);
```

Reply:
@ckerr commit replaced as you suggested :)